.github/workflows/linux.yml

```diff
@@ -99,7 +99,7 @@ jobs:
         run: python3 -m pip install -r ${OPENVINO_REPO}/src/bindings/python/wheel/requirements-dev.txt
 
       - name: Setup ccache
-        uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
+        uses: actions/cache@v4
```

Comment:
hashes must be used

Reply:
Added.
What is a reason to add hash? Is something not working?

Reply:
to prevent automatic updates - tags can be overridden by external repo owners and it's not safe to automatically use new 3rd party code.